app/views/scans/_ineligible_exception.html.erb

```diff
@@ -0,0 +1,5 @@
+<strong class="h3">Scan-to-PDF not available.</strong>
+
+<p>
+This pilot service is currently available only to faculty, grad students, postdocs, academic staff, fellows, and visiting scholars. Your feedback will help us refine and expand the service!
+</p>
```

Comment:
We need to do that silly flash[:html_safe] thing due to the way flash messages are stored across requests (which loses the html_safe bit). I think this needs to be a partial, not an i18n string, when we convert `"Your feedback"` into a link as designed.